src/traffic_server/traffic_server.cc

```diff
@@ -1957,7 +1957,14 @@ main(int /* argc ATS_UNUSED */, const char **argv)
     RecConfigWarnIfUnregistered();
 
     // "Task" processor, possibly with its own set of task threads
-    tasksProcessor.start(num_task_threads, stacksize);
+    {
+      tasksProcessor.start(num_task_threads, stacksize);
```

Comment:
How confident are you that the task threads have actually started by the time the `start` method returns? I thought there was some sort of counter on the threads to track that.